src/main/java/org/ladocuploader/app/cli/SubmissionTransfer.java

```diff
@@ -74,35 +74,51 @@ public SubmissionTransfer(SubmissionRepository submissionRepository, Transmissio
     this.ftpsClient = ftpsClient;
   }
 
+  @ShellMethod(key = "transferSubmissions")
   public void transferSubmissions() {
     // Get submissions to transfer
     String batchSeq = Long.toString(transmissionRepository.nextValueBatchSequence());
     String batchIndex = Strings.padStart(batchSeq, BATCH_INDEX_LEN, '0');
+    UUID uuid = UUID.randomUUID();
```

Comment:
Saw this and it made me curious about UUID collisions. Quick research suggest that it is extremely improbable and therefore unlikely.

Reply:
Yeah - unlikely and also ok if theres collisions, ideally I'd like to track "batchIndex" in the "run_id" column, but since it's restricted to UUID, Im just generating a new one here and logging the batchIndex+runId